.ci/download-deps.sh

```diff
@@ -24,7 +24,7 @@ function get_deps() {
 	if [ ! -d tpm2-abrmd ]; then
 		echo "pwd clone abrmd: `pwd`"
 		git clone --depth=1 \
-		--branch 2.2.0 https://github.com/tpm2-software/tpm2-abrmd.git
+		--branch master https://github.com/tpm2-software/tpm2-abrmd.git
```

Comment:
should we change this? I think we should only bump this as we need features....

Reply:
It would also be nice to have a TSS_VERSION and ABRMD_VERSION variable in .travis.yaml file that we can set to the appropriate values as we need.

Reply:
> It would also be nice to have a TSS_VERSION and ABRMD_VERSION variable in .travis.yaml file that we can set to the appropriate values as we need.

True. I will add the variable.